File: src/TrainingBundle/Entity/Entrainement.php
<?php

namespace TrainingBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * Entrainement
 *
 * @ORM\Table(name="entrainement")
 * @ORM\Entity(repositoryClass="TrainingBundle\Repository\EntrainementRepository")
 */
class Entrainement
{
    /**
     * @ORM\ManyToOne(targetEntity="UserBundle\Entity\User")
     * @ORM\JoinColumn(name="userId", referencedColumnName="id")
     */
    private $user;

    /**
     * @ORM\ManyToOne(targetEntity="UserBundle\Entity\User")
     * @ORM\JoinColumn(name="entraineurId", referencedColumnName="id")
     */
    private $entraineur;

    /**
     * @ORM\ManyToOne(targetEntity="Animal")
     * @ORM\JoinColumn(name="animalId", referencedColumnName="id")
     */
    private $animal;

    /**
     * @ORM\ManyToOne(targetEntity="ProductBundle\Entity\Produit")
     * @ORM\JoinColumn(name="produitId", referencedColumnName="id")
     */
    private $produit;

    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="categorie", type="string", length=255)
     */
    private $categorie;

    /**
     * @var int
     *
     * @ORM\Column(name="nbHeures", type="integer")
     */
    private $nbHeures;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateEnt", type="date",nullable=true)
     */
    private $dateEnt;

    /**
     * @var float
     *
     * @ORM\Column(name="prix", type="float")
     */
    private $prix;

    /**
     * @var string
     *
     * @ORM\Column(name="lieu", type="string", length=255)
     * @Assert\NotBlank(message="lieu ne doit pas etre vide")
     */
    private $lieu;

    /**
     * @var string
     *
     * @ORM\Column(name="accepter", type="string", length=255 ,options={"default": "encours"},nullable=true)
     */
    private $accepter;

    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set categorie
     *
     * @param string $categorie
     *
     * @return Entrainement
     */
    public function setCategorie($categorie)
    {
        $this->categorie = $categorie;

        return $this;
    }

    /**
     * Get categorie
     *
     * @return string
     */
    public function getCategorie()
    {
        return $this->categorie;
    }

    /**
     * Set nbHeures
     *
     * @param integer $nbHeures
     *
     * @return Entrainement
     */
    public function setNbHeures($nbHeures)
    {
        $this->nbHeures = $nbHeures;

        return $this;
    }

    /**
     * Get nbHeures
     *
     * @return int
     */
    public function getNbHeures()
    {
        return $this->nbHeures;
    }

    /**
     * Set dateEnt
     *
     * @param \DateTime $dateEnt
     *
     * @return Entrainement
     */
    public function setDateEnt($dateEnt)
    {
        $this->dateEnt = $dateEnt;

        return $this;
    }

    /**
     * Get dateEnt
     *
     * @return \DateTime
     */
    public function getDateEnt()
    {
        return $this->dateEnt;
    }

    /**
     * Set prix
     *
     * @param float $prix
     *
     * @return Entrainement
     */
    public function setPrix($prix)
    {
        $this->prix = $prix;

        return $this;
    }

    /**
     * Get prix
     *
     * @return float
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * Set lieu
     *
     * @param string $lieu
     *
     * @return Entrainement
     */
    public function setLieu($lieu)
    {
        $this->lieu = $lieu;

        return $this;
    }

    /**
     * Get lieu
     *
     * @return string
     */
    public function getLieu()
    {
        return $this->lieu;
    }

    /**
     * @return mixed
     */
    public function getUser()
    {
        return $this->user;
    }

    /**
     * @param mixed $user
     */
    public function setUser($user)
    {
        $this->user = $user;
    }

    /**
     * @return mixed
     */
    public function getEntraineur()
    {
        return $this->entraineur;
    }

    /**
     * @param mixed $entraineur
     */
    public function setEntraineur($entraineur)
    {
        $this->entraineur = $entraineur;
    }

    /**
     * @return mixed
     */
    public function getAnimal()
    {
        return $this->animal;
    }

    /**
     * @param mixed $animal
     */
    public function setAnimal($animal)
    {
        $this->animal = $animal;
    }

    /**
     * @return mixed
     */
    public function getProduit()
    {
        return $this->produit;
    }

    /**
     * @param mixed $produit
     */
    public function setProduit($produit)
    {
        $this->produit = $produit;
    }

    /**
     * @return string
     */
    public function getAccepter()
    {
        return $this->accepter;
    }

    /**
     * @param string $accepter
     */
    public function setAccepter($accepter)
    {
        $this->accepter = $accepter;
    }


}

File: web/java/huntkingdom/src/Entities/Entrainement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.util.Date;

/**
 *
 * @author tibh
 */
public class Entrainement {

    private int id;
    private String categorie;
    private int nbHeures;
    private Date dateEnt;
    private Double prix;
    private String lieu;
    private int userId;
    private int entraineurId;
    private int animalId;
    private int produitId;
    private String accepter;
    
    public Entrainement(){}
    public Entrainement(int id, String categorie, int nbHeures, Date dateEnt, Double prix, String lieu, int userId, int entraineurId, int animalId, int produitId, String accepter) {
        this.id = id;
        this.categorie = categorie;
        this.nbHeures = nbHeures;
        this.dateEnt = dateEnt;
        this.prix = prix;
        this.lieu = lieu;
        this.userId = userId;
        this.entraineurId = entraineurId;
        this.animalId = animalId;
        this.produitId = produitId;
        this.accepter = accepter;
    }
    
    public Entrainement(String categorie, int nbHeures, Date dateEnt, Double prix, String lieu,int userId ,int animalId,int produitId  )
    {
        this.id=0;
        this.categorie = categorie;
        this.nbHeures = nbHeures;
        this.dateEnt = dateEnt;
        this.prix = prix;
        this.lieu = lieu;
        this.userId = userId;
        this.entraineurId = 0;
        this.animalId = animalId;
        this.produitId = produitId;
        this.accepter = "encours";
        
        
    
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getCategorie() {
        return categorie;
    }

    public void setCategorie(String categorie) {
        this.categorie = categorie;
    }

    public int getNbHeures() {
        return nbHeures;
    }

    public void setNbHeures(int nbHeures) {
        this.nbHeures = nbHeures;
    }

    public Date getDateEnt() {
        return dateEnt;
    }

    public void setDateEnt(Date dateEnt) {
        this.dateEnt = dateEnt;
    }

    public Double getPrix() {
        return prix;
    }

    public void setPrix(Double prix) {
        this.prix = prix;
    }

    public String getLieu() {
        return lieu;
    }

    public void setLieu(String lieu) {
        this.lieu = lieu;
    }

    public int getUserId() {
        return userId;
    }

    public void setUserId(int userId) {
        this.userId = userId;
    }

    public int getEntraineurId() {
        return entraineurId;
    }

    public void setEntraineurId(int entraineurId) {
        this.entraineurId = entraineurId;
    }

    public int getAnimalId() {
        return animalId;
    }

    public void setAnimalId(int animalId) {
        this.animalId = animalId;
    }

    public int getProduitId() {
        return produitId;
    }

    public void setProduitId(int produitId) {
        this.produitId = produitId;
    }

    public String getAccepter() {
        return accepter;
    }

    public void setAccepter(String accepter) {
        this.accepter = accepter;
    }

    @Override
    public String toString() {
        return "Entrainement{" + "id=" + id + ", categorie=" + categorie + ", nbHeures=" + nbHeures + ", dateEnt=" + dateEnt + ", prix=" + prix + ", lieu=" + lieu + ", userId=" + userId + ", entraineurId=" + entraineurId + ", animalId=" + animalId + ", produitId=" + produitId + ", accepter=" + accepter + '}';
    }
    

}
